pkgs/by-name/in/install-grub-ng/src/main.rs

```diff
@@ -0,0 +1,38 @@
+mod builder;
+mod config;
+mod grub;
+
+use std::path::Path;
+
+use eyre::{Result, bail};
+use roxmltree::Document;
+
+use crate::{builder::Builder, config::Config};
+
+fn main() -> Result<()> {
+    let mut args = std::env::args();
+    let Some(config_file) = args.nth(1) else {
```

Comment:
style nit: For easier changing:
```suggestion
    let mut args = std::env::args().skip(1);

    let Some(config_file) = args.next() else {
```